crates/solidity/testing/perf/benches/iai/dataset.rs

```diff
@@ -243,3 +246,55 @@ pub fn run_query(trees: &Vec<Node>) {
         "Function names don't match: {results:#?}"
     );
 }
+
+struct NoOpResolver;
+
+impl PathResolver for NoOpResolver {
+    fn resolve_path(&self, _context_path: &str, path_to_resolve: &str) -> Option<String> {
+        Some(path_to_resolve.to_string())
+    }
+}
+
+pub fn run_create_bindings() {
+    let _ = bindings::create_with_resolver(SOLC_VERSION, Arc::new(NoOpResolver {}));
+}
+
+pub fn run_bindings(trees: &Vec<Node>) -> Vec<Bindings> {
+    let mut result = vec![];
+    let mut definition_count = 0_usize;
+
+    for tree in trees {
+        let mut bindings = bindings::create_with_resolver(SOLC_VERSION, Arc::new(NoOpResolver {}));
```

Comment:
I wonder why are we creating multiple `Bindings` objects, rather than a single one that binds all files?
Wouldn't that be a closer scenario to what the users will actually do?

Reply:
TBH, I thought the input files were unrelated to each other, so it'd make sense to treat them as separate "projects", hence the separate bindings objects.

Would it make sense to load them and identify them with proper paths so that some of the imports inside can resolve correctly? Eg. I see `Governor.sol` imports `../utils/math/SafeCast.sol` which is another of the files in the set.